snow/engine/avalanche/transitive.go

```diff
@@ -259,15 +261,19 @@ func (t *Transitive) PullQuery(vdr ids.ShortID, requestID uint32, vtxID ids.ID)
 		errs:      &t.errs,
 	}
 
+	// Clear abandoned vertices before issuing anything
+	t.abandonedVertices = false
```

Comment:
Can we move this to the start of `issueFromByID`?
We always call it right before calling that method.

Reply:
oh nvm I see why you can't now